i18n/en.pot

```diff
@@ -92,14 +92,8 @@ msgstr ""
 msgid "Print"
 msgstr ""
 
-msgid "There was an error loading data for this item"
-msgstr "There was an error loading data for this item"
-
-msgid "Open this item in {{appName}}"
-msgstr "Open this item in {{appName}}"
-
-msgid "Visualizations"
```

Comment:
Do you know the source of these translation changes?

Reply:
No, but the last revision date is 2021-06-10 so I assume its another commit between then and now. From the commit history it looks like most of them are actually yours, so maybe you recognise the strings?